aspnet/getting-started/installing-on-linux.rst

```diff
@@ -24,24 +20,50 @@ The rest of this section deals with setting up a machine to run applications wit
 Installing on Debian, Ubuntu and derivatives
 --------------------------------------------
 
-The following instructions were tested using Ubuntu 14.04 and Mint 17.01
+The following instructions were tested using Ubuntu 14.04.
+
+Install the .NET Version Manager (DNVM)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on Linux.
+
+To install DNVM run the following::
+
+    curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
```

Comment:
Should there be something about "permanently" sourcing dnvm, i.e. putting it in .bashrc or .bash_profile